components/project-card/project-card.scss

```diff
@@ -146,10 +140,8 @@
 // >= 768px
 @media screen and (min-width: $bp-md) {
   .project-card {
-    @include project-card-width(3);
-
     &.detail-view {
-      @include project-card-width(2);
+      max-width: 600px;
```

Comment:
Any reason you wouldn't use a column mixin (or class on the element itself) so we're consistently using the grid? (Thinking `col-12 col-md-8` or something like that comes close to 600px)